### ISSUE TITLE
Qt5: dump_tree() cannot list "detouched" children of main window for Dolphin Emulator, but Inspect.exe can

### ISSUE THREAD

**Issue by vasily-v-ryabov** [pywinauto/pywinauto#644]
Found on StackOverflow: https://stackoverflow.com/q/54003752/3648361

There is Dolphin Emulator downloads: https://dolphin-emu.org/download/

Latest version 5.0-9281 has interesting app side issue that Inspect.exe can workaround. Main window property "Children" getter returns this error (listed in Inspect.exe):
```
[Error: calling getter for this property: hr=0xFFFFFFFF80004005 - Unspecified error]
```
But Inspect.exe shows the hierarchy with "[+]" button anyway. This would be interesting to investigate and apply the same workaround in pywinauto.

**Comment by vasily-v-ryabov** [comment]
I think we can close this issue after enabling optional way to use `FindAll()`.

**Comment by eltimen** [comment]
> ```
> [Error: calling getter for this property: hr=0xFFFFFFFF80004005 - Unspecified error]
> ```
I tried to reproduce this, and the above error message doesn't show in my environment (Windows 7 x64, UIAutomationCore.dll version 7.0.0.0).

During debugging the `print_control_identifiers()` problem, I found out that COM method `IUIAutomationElement::FindAll` raises `COMError (-2147024882, 'Not enough memory resources are available to complete this operation.', (None, None, None, 0, None))`. Details known to me:
1. This problem only occurs when FindAll called with `TreeScope_Descendants` (inside `print_control_identifiers()` and `descendants()`), but `children()` method works fine.
2. The problem is probably not related to the width or depth of the widget tree (because it works fine with Qt-based benchmark with 62 nested widgets).
3. The thread `UIAutomationCore.dll!UiaHTextRangeFromVariant` is created inside `Dolphin.exe` process after FindAll call and causes high CPU usage. Probably, the root of the problem can be found by examining stack of this thread: <details> <summary>Screenshot</summary>
![screen](https://user-images.githubusercontent.com/53598853/98091986-36031a00-1e97-11eb-9f1b-e15c2aee2839.jpg)
</details>

I have not found other cases where this bug appears, besides Dolphin. But the problem is solved after replacing `FindAll` with `RawViewWalker`.
